terraso_backend/apps/graphql/schema/sites.py

```diff
@@ -59,6 +60,7 @@ class SiteNode(DjangoObjectType):
     soil_data = graphene.Field(
         "apps.soil_id.graphql.soil_data.SoilDataNode", required=True, default_value=SoilData()
     )
+    notes = graphene.List(SiteNoteNode)
```

Comment:
Issue: Graphene should be smart enough to define this field automatically if you add 'notes' in the fields list in line 68